conn.go

```diff
@@ -1161,7 +1190,7 @@ func (c *Conn) SetPingHandler(h func(appData string) error) {
 			err := c.WriteControl(PongMessage, []byte(message), time.Now().Add(writeWait))
 			if err == ErrCloseSent {
 				return nil
-			} else if e, ok := err.(net.Error); ok && e.Temporary() {
+			} else if _, ok := err.(net.Error); ok {
```

Comment:
In practice, the original code returned nil for deadline exceeded errors.   The new code returns nil for all network errors. Change to detect [deadline exceeded errors](https://pkg.go.dev/os#ErrDeadlineExceeded) only.